tests/slsa_analyzer/test_slsa_requirements.py

```diff
@@ -7,41 +7,24 @@
 
 from unittest import TestCase
 
-from macaron.slsa_analyzer.levels import SLSALevels
-from macaron.slsa_analyzer.slsa_req import BUILD_REQ_DESC, Category, SLSAReq, get_requirements_dict
+from macaron.slsa_analyzer.slsa_req import SLSAReqStatus
 
 
 class TestSLSARequirements(TestCase):
```

Comment:
It's probably a good idea to also convert this `unittest` class into single test functions since we are moving away from `unittest`.

Reply:
Done.